server/channels/app/webhook.go

```diff
@@ -103,7 +103,7 @@ func (a *App) TriggerWebhook(c request.CTX, payload *model.OutgoingWebhookPayloa
 		contentType = "application/json"
 		jsonBytes, err = json.Marshal(payload)
 		if err != nil {
-			c.Logger().Warn("Failed to encode to JSON", mlog.Err(err))
+			c.Logger().Warn("Failed to encode to JSON", mlog.String("outgoing_webhook_id", hook.Id), mlog.Err(err))
```

Comment:
I wonder if it makes sense to use for this function something like `c.Logger().With(...)`. That way, we only add the id in the with, and then we can use the newly generated logger everywhere.

0/5